core/domain/services/graphql/connection_resolvers/AbstractConnectionResolver.php

```diff
@@ -3,6 +3,7 @@
 namespace EventEspresso\core\domain\services\graphql\connection_resolvers;
 
 use WPGraphQL\Data\Connection\AbstractConnectionResolver as WPGraphQLConnectionResolver;
+use GraphQLRelay\Relay;
```

Comment:
whaaaaaaaaaaaat?

isn't that redundant because of Apollo? or was this added because it has the logic needed for UUIDs?

Reply:
That package is bundled with WPGraphQL and it uses that package to generate those IDs, it's a better option than UUIDs as the Relay package is managed by Facebook and works well with GraphQL.